src/SignalR/clients/ts/signalr/src/FetchHttpClient.ts

```diff
@@ -91,7 +91,7 @@ export class FetchHttpClient extends HttpClient {
                 },
                 method: request.method!,
                 mode: "cors",
-                redirect: "manual",
+                redirect: "follow",
```

Comment:
This is the default. You could not specify it instead. Same for `mode: "cors"`

Reply:
I prefer to be explicit with these options. The "mode" default is different depending on how you instantiate the request object, plus we use a fetch polyfill for Node apps and it could have different defaults.

Reply:
I'm fine with keeping it explicit.